docker/debian8/Dockerfile-kraken-opentelemetry

```
@@ -0,0 +1,13 @@
+ARG KRAKEN_TAG
+FROM navitia/kraken:${KRAKEN_TAG}
+
+## see https://opentelemetry.io/docs/collector/getting-started/#deb-installation
+RUN apt-get update && apt-get install -y wget && rm -rf /var/lib/apt/lists/*
+RUN wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.73.0/otelcol-contrib_0.73.0_linux_amd64.deb
```

On debian8, `curl` works better:
```suggestion
RUN apt-get update && apt-get install -y curl && rm -rf /var/lib/apt/lists/*
RUN curl https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.73.0/otelcol-contrib_0.73.0_linux_amd64.deb -O -L
```